HousingSearchApi.Tests/V1/UseCase/IGetPersonListUseCase.cs

```diff
@@ -0,0 +1,10 @@
+using System.Threading.Tasks;
+using HousingSearchApi.V1.Domain;
+
+namespace HousingSearchApi.Tests.V1.UseCase
+{
+    public interface IGetPersonListUseCase
```

Comment:
This looks like an implementation file, not a test file. Should it be here?

Reply:
Ah I see. Yes, I'll remove them. They got left here as I was moving things around (tbh I didn't look that much into any of this in this ticket, just wanting to merge it to master so I have a semi good looking base from where to branch ! Good call though, removing them in present ticket)